### PR TITLE
fix up containsPoint for trimmed sprite

### PR DESCRIPTION
**Pull request by GoodBoyDigital**
### Summary
🐛✅🧹

This pull request improves the `SpriteView` class and its tests. It fixes some bugs and edge cases related to trimmed textures and bounds calculations. It also removes some unused code from the `layer-tint.scene.ts` file.

> _Sing, O Muse, of the skillful coder who refined the SpriteView class_
> _And made it work with trimmed textures, a feat that none surpassed._
> _He fixed the imports and the tests, and cleared the layer-tint scene_
> _Of unused code and parameters, and made it run more clean._

### Walkthrough
*  Simplify and fix `SpriteView` methods for trimmed textures ([link](https://github.com/pixijs/pixijs/pull/9882/files?diff=unified&w=0#diff-ec873ce7abacc5136e469e35fdd0dfccda62949329abfb67727519fd6b2a63e5L85-R89), [link](https://github.com/pixijs/pixijs/pull/9882/files?diff=unified&w=0#diff-ec873ce7abacc5136e469e35fdd0dfccda62949329abfb67727519fd6b2a63e5L105-R102), [link](https://github.com/pixijs/pixijs/pull/9882/files?diff=unified&w=0#diff-ec873ce7abacc5136e469e35fdd0dfccda62949329abfb67727519fd6b2a63e5L153-R141), [link](https://github.com/pixijs/pixijs/pull/9882/files?diff=unified&w=0#diff-0a8b750c0adec1b3abe9459f95441b75c1d3297e2ab9b3d33ff93dd539cf49b9R213-R253), [link](https://github.com/pixijs/pixijs/pull/9882/files?diff=unified&w=0#diff-07fc4c1603a5e6747eaef03ec8b7a516baf7d114b55913d2d1c92726a5a43bb9R251-R272))
* Remove unused imports and parameters from test files ([link](https://github.com/pixijs/pixijs/pull/9882/files?diff=unified&w=0#diff-0a8b750c0adec1b3abe9459f95441b75c1d3297e2ab9b3d33ff93dd539cf49b9L2-R4), [link](https://github.com/pixijs/pixijs/pull/9882/files?diff=unified&w=0#diff-07fc4c1603a5e6747eaef03ec8b7a516baf7d114b55913d2d1c92726a5a43bb9L1-R9), [link](https://github.com/pixijs/pixijs/pull/9882/files?diff=unified&w=0#diff-674a5379865dbea4d25bff8a6b0332c6c3f9634729c096d60c09f3cac47723e4L4-R4), [link](https://github.com/pixijs/pixijs/pull/9882/files?diff=unified&w=0#diff-674a5379865dbea4d25bff8a6b0332c6c3f9634729c096d60c09f3cac47723e4L12-R10))

### CODE REVIEW

File: tests/renderering/sprite/Sprite.test.ts [hunk 1]
```diff
@@ -208,6 +210,47 @@ describe('Sprite', () =>
 
             expect(sprite.view.containsPoint(point)).toBe(false);
         });
+
+        it.only('should return true when point inside a trimmed sprite', () =>
```
Comment (achamas-playco) [inline tests/renderering/sprite/Sprite.test.ts:214]:
```suggestion
        it('should return true when point inside a trimmed sprite', () =>
```

File: tests/visual/scenes/scene/layer-tint.scene.ts [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
 import { Graphics } from '../../../../src/scene/graphics/shared/Graphics';
 import { GraphicsContext } from '../../../../src/scene/graphics/shared/GraphicsContext';
-import { Text } from '../../../../src/scene/text/Text';
 
-import type { Renderer } from '../../../../src/rendering/renderers/types';
 import type { TestScene } from '../../types';
 
 export const scene: TestScene = {
     it: 'should tint layers correctly',
     only: true,
```
Comment (achamas-playco) [inline tests/visual/scenes/scene/layer-tint.scene.ts:9]:
```suggestion
```

**Review by Zyie — APPROVED** [review]
